File: 프로그래머스/2/148653. 마법의 엘리베이터/마법의 엘리베이터.py
def solution(storey):
    count = 0
    while storey:
        current = storey % 10
        next_ = (storey//10)%10
        if current<5 or (current==5 and next_<5):
            count += current
            storey = storey//10
        else:
            count += (10-current)
            storey = storey//10+1
            
    return count

File: 프로그래머스/2/148653. 마법의 엘리베이터/마법의 엘리베이터.js
function solution(storey) {
    let count = 0;
    while(storey){
        let current = storey%10;
        let next = Math.floor(storey/10)%10;
        if(current<5 || (current===5 && next<5)){
            count += current;
            storey = Math.floor(storey / 10);
        } else{
            count += (10-current);
            storey = Math.floor(storey / 10) + 1;
        }
    }
    return count;
}